recipe/meta.yaml

```diff
@@ -28,7 +28,7 @@ test:
 
 about:
   home: https://github.com/timothycrosley/isort
-  license: MIT License
+  license: MIT
```

Comment:
FYI this will result in a change to the Readme after re-rendering. Not that it needs to be dealt with in this PR per se. Just noting the change.